sentry-debug-images/src/lib.rs

```diff
@@ -1,20 +1,23 @@
 //! The Sentry Debug Images Integration.
 //!
-//! The `DebugImagesIntegration` adds metadata about the loaded shared libraries
-//! to Sentry `Event`s.
+//! The [`DebugImagesIntegration`] adds metadata about the loaded shared
+//! libraries to Sentry [`Event`]s.
+//!
+//! This Integration only works on Unix-like OSs right now. Support for Windows
+//! will be added in the future.
 //!
 //! # Configuration
 //!
-//! The integration by default attaches this information to all Events, but a
-//! custom filter can be defined as well.
+//! The integration by default attaches this information to all [`Event`]s, but
+//! a custom filter can be defined as well.
 //!
 //! ```
 //! use sentry_core::Level;
-//! let integration = sentry_debug_images::DebugImagesIntegration {
-//!     filter: Box::new(|event| event.level >= Level::Warning),
-//!     ..Default::default()
-//! };
+//! let integration = sentry_debug_images::DebugImagesIntegration::new()
+//!     .filter(|event| event.level >= Level::Warning);
 //! ```
+//!
+//! [`Event`]: sentry_core::Event
```

Comment:
This link does not actually work. If I `cargo doc` and click it, it sends me to: `sentry_debug_images/sentry_core::Event`. Can you double check this and other links that we have?

Reply:
I compiled with nightly. The other link for `DebugImagesIntegration` does work.

Reply:
```
warning: unresolved link to `sentry_core::Event`
  --> sentry-debug-images\src\lib.rs:20:16
   |
20 | //! [`Event`]: sentry_core::Event
   |                ^^^^^^^^^^^^^^^^^^ the crate `sentry_core` has no inner item named `Event`
   |
   = note: `#[warn(broken_intra_doc_links)]` on by default
```

Wow, this is awesome!

Also, I fixed some imports in sentry-core which error when using `no-default-features`, which apparently happens when you document a single one of these crates, lol.